force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -110,7 +149,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return The grade for the given score.
      */
     public static String determineGrade(Integer score) {
-        return null; // Replace null with the variable you used to store the result
+        if (score >= 90) {
```

Comment:
(s) Solid logic. This is a good use case for the other kind of flow control - a `switch when` statement